tests/Unit/AbstractDiffVendorCommandTest.php

```
@@ -0,0 +1,273 @@
+    {
+        $command = $this->getTestCommand();
+
+        // Simulate a Windows path
+        $windowsBasePath = 'C:\\Users\\test\\project';
+        $windowsAbsolutePath = 'C:\\Users\\test\\project\\config\\app.php';
+
+        // Mock base_path temporarily (we can't easily change it, so we test the normalization)
+        $normalizedPath = $command->exposeNormalizePath($windowsAbsolutePath);
+
+        $this->assertStringNotContainsString('\\', $normalizedPath);
+    }
```

PHPMD (2.15.0)

The method test_to_relative_path_handles_windows_paths is not named in camelCase. (undefined)

(CamelCaseMethodName)